### PR TITLE
Add Disclaimer Link

### PR DESCRIPTION
**Pull request by adamrusted**
Adds a link in the README to the Disclaimer.md file in the main repo.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -56,6 +56,8 @@ You can also [download the latest version of the package][latest-release] and co
 
 ## Usage
 
+> :information_source: We ask that all users read our [legal disclaimer](https://github.com/simple-icons/simple-icons/blob/develop/DISCLAIMER.md) before using icons from Simple Icons.
```
Comment (ericcornelissen) [inline README.md:59]: I think the link should be to `master` now that the `DISCLAIMER.md` file available there

```suggestion
> :information_source: We ask that all users read our [legal disclaimer](https://github.com/simple-icons/simple-icons/blob/master/DISCLAIMER.md) before using icons from Simple Icons.
```

**Review by ericcornelissen — COMMENTED** [review]
I think it should be placed higher up in the README so that it is harder to miss. Unfortunately it will be a bit different from the main repository but maybe we can put it after [the Setup title](https://github.com/simple-icons/simple-icons-font#setup)?

Alternatively we can put it before the first title in the README of all repos, that way it would be more consistent...

**Comment by adamrusted** [comment]
> Alternatively we can put it before the first title in the README of all repos, that way it would be more consistent...

I like that idea. In theory, the notice won't have to be there forever (I imagine it can be demoted to a footnote later down the line) and keeps it top priority during these early v5 releases.

**Comment by ericcornelissen** [comment]
I'd say we should always keep it at the top, but I guess that's a separate discussion that doesn't impact this Pull Request.

**Comment by ericcornelissen** [comment]
If we do decide to move it before the first title, I think we should do that consistently across all our projects. As such I think we need approval from other @simple-icons/maintainers on this one before we merge this in.

I'd personally be in favor of this. One suggestion I have is to put a horizontal divider between the centered part and the disclaimer notice.

File: README.md [hunk 2]
```diff
@@ -13,6 +13,8 @@ Free SVG icon font for popular brands. See them all on one page at <a href="http
 <a href="https://packagist.org/packages/simple-icons/simple-icons-font"><img src="https://img.shields.io/packagist/v/simple-icons/simple-icons-font?logo=packagist&logoColor=white" alt="Build status" /></a>
 </p>
 
+> :information_source: We ask that all users read our [legal disclaimer](https://github.com/simple-icons/simple-icons/blob/master/DISCLAIMER.md) before using icons from Simple Icons.
+
 ## Setup
```
Comment (ericcornelissen) [inline README.md:19]: In the meantime, I'd say changing it to this :arrow_down: makes it consistent with the main repository. That way we can merge this as is before the next release for sure, and look into moving it up later.

Agreed @adamrusted?

```suggestion
## Setup

> :information_source: We ask that all users read our [legal disclaimer](https://github.com/simple-icons/simple-icons/blob/master/DISCLAIMER.md) before using icons from Simple Icons.
```

File: README.md [hunk 3]
```diff
@@ -15,6 +15,7 @@ Free SVG icon font for popular brands. See them all on one page at <a href="http
 
 ## Setup
 
+> :information_source: We ask that all users read our [legal disclaimer](https://github.com/simple-icons/simple-icons/blob/master/DISCLAIMER.md) before using icons from Simple Icons.
 ### CDN Setup
```
Comment (ericcornelissen) [inline README.md:19]:
```suggestion
> :information_source: We ask that all users read our [legal disclaimer](https://github.com/simple-icons/simple-icons/blob/master/DISCLAIMER.md) before using icons from Simple Icons.

### CDN Setup
```

**Review by ericcornelissen — APPROVED** [review]
I'll be merging this in for now so at least some link is included in the next release of the package. Again, I'm happy to reconsider to position of the notice :slightly_smiling_face:

**Pull request merged**